### ISSUE TITLE
method_deprecated

### ISSUE THREAD

**Issue by avdhil** [FlyingDiver/Indigo-Slack2#2]
I just installed the plugin. Config should be fine, but I do get an error message:

```
 Started plugin "Slack 2 1.0.1"
   Slack 2 Debug                   Slack 2 startup
   Slack 2 Debug                   Slack Mi-Casa: Starting Device
   Slack 2 Error                   Slack Mi-Casa: Slack connection error: method_deprecated
```

**Comment by FlyingDiver** [comment]
Cannot reproduce.  Mine is working fine, same plugin version.

Get a new OAuth token from the website and try again.

**Comment by avdhil** [comment]
Thanks! It seems like I'm lost while creating the apps at api.slack.com/apps.
The line in your manual 'Add "Send messages as Permission Scope"' does not
match with mine. I can only define Bot token scopes and User token scopes.
In there I can't find the right scopes.

What is it that I'm doing wrong? Help is appreciated!

--Alexander

**Comment by FlyingDiver** [comment]
They've changed the website around.  Let me go through it again and update the instructions.

**Comment by FlyingDiver** [comment]
Ok, I see the problem.  The method for getting the list of available channels has changed, and isn't allowed for apps created after June of this year.  See https://api.slack.com/methods/channels.list

I'll need to figure out how to use the new methods.

**Comment by FlyingDiver** [comment]
Try https://github.com/FlyingDiver/Indigo-Slack2/releases/tag/1.1.0

You need the following User Token Scopes:
channels:read
chat:write
files:write

Then use the "OAuth Access Token" (starts with "xoxp-")

**Comment by avdhil** [comment]
That works. Perfect! Thank you so much!

Kind regards,
Alexander

**Comment by FlyingDiver** [comment]
Fixed in v1.1.0.
